.classpath

```diff
@@ -46,5 +46,7 @@
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
 	<classpathentry kind="lib" path="lib/commons-codec-1.9.jar"/>
 	<classpathentry kind="lib" path="lib/sqlite-jdbc-3.8.11.1.jar"/>
+	<classpathentry kind="lib" path="lib/generex-0.0.5-SNAPSHOT.jar"/>
```

Comment:
As noted in the commit message that's a custom build to support the fuzz add-on requirements (that's one of the reasons it has SNAPSHOT).
I'd not expect any maintainability issues other than update the fuzz add-on, since it's a self contained dependency (it's true that add-ons that depend on fuzz add-on could use it but, very unlikely that will happen and it's not something that's advertised as supported).